platforms/Windows/SideBySideUpgradeStrategy.props

```diff
@@ -18,9 +18,13 @@
 
   <PropertyGroup>
     <BldUpgradeCode>{7E95DC06-7F84-4E8E-A038-8304AF0468FB}</BldUpgradeCode>
```

Comment:
We should rename these to indicate the `Asserts` variant.

Reply:
changed